packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -39,45 +39,77 @@ export const chatMessageStyles: ComponentSlotStylesPrepared<ChatMessageStylesPro
+              height: `calc(100% - ${v.paddingCompact} - ${v.paddingCompact})`,
+              left: pxToRem(-56),
+              top: v.paddingCompact,
+              width: pxToRem(2),
+            }
+          : {
+              borderBottomLeftRadius: 'inherit',
+              borderTopLeftRadius: 'inherit',
+              height: '100%',
+              left: '0',
+              top: '0',
+              width: pxToRem(3),
+            }),
+      },
+    }),
+
+    ...getBorderFocusStyles({ variables: siteVariables }),
 
-      wordBreak: 'break-word',
-      wordWrap: 'break-word',
-
-      ...((v.hasMention || v.isImportant) && {
-        '::before': {
-          content: '""',
-          backgroundColor: v.hasMention ? v.hasMentionColor : v.isImportantColor,
-          height: '100%',
-          left: '0',
-          position: 'absolute',
-          top: '0',
-          width: pxToRem(3),
-          borderBottomLeftRadius: 'inherit',
-          borderTopLeftRadius: 'inherit',
+    ...(p.compact && {
+      width: '100%',
+      ...(p.focused && {
+        backgroundColor: v.compactHoverBackground,
+        border: v.compactHoverBorder,
+        [`& .${chatMessageSlotClassNames.timestamp}`]: {
+          opacity: 1,
         },
       }),
+      '&:hover': {
+        backgroundColor: v.compactHoverBackground,
+        border: v.compactHoverBorder,
+        [`& .${chatMessageSlotClassNames.timestamp}`]: {
+          opacity: 1,
+        },
+      },
+      padding: v.paddingCompact,
+    }),
 
-      ...getBorderFocusStyles({ variables: siteVariables }),
-
-      // actions menu's appearance can be controlled by the value of showActionMenu variable - in this
-      // case this variable will serve the single source of truth on whether actions menu should be shown.
-      // Otherwise, if the variable is not provided, the default appearance logic will be used for actions menu.
-      ...(_.isNil(v.showActionMenu) &&
-        p.hasActionMenu && {
-          ':hover': {
-            [`> .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
-          },
-          ...(p.showActionMenu && {
-            [`> .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
-          }),
+    // actions menu's appearance can be controlled by the value of showActionMenu variable - in this
+    // case this variable will serve the single source of truth on whether actions menu should be shown.
+    // Otherwise, if the variable is not provided, the default appearance logic will be used for actions menu.
+    ...(_.isNil(v.showActionMenu) &&
+      p.hasActionMenu && {
+        ':hover': {
+          [`> .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
+        },
+        ...(p.showActionMenu && {
+          [`> .${chatMessageSlotClassNames.actionMenu}`]: displayActionMenu(v.overlayZIndex),
         }),
+      }),
 
+    ...(!p.compact && {
       ...(p.attached === true && {
```

Comment:
shouldn't this just be
```suggestion
    ...(!p.compact && p.attached === true && {
```

Reply:
No. The `!p.compact` applies to all three `attached` options:
![image](https://user-images.githubusercontent.com/2564094/120026098-6c74c380-bfa6-11eb-97a9-e354ea8c07da.png)